src/managementpartner/azext_managementpartner/commands.py

```diff
@@ -4,22 +4,14 @@
 # --------------------------------------------------------------------------------------------
 
 
-from azure.cli.core.commands import CliCommandType
-from ._client_factory import managementpartner_partner_client_factory
-from ._exception_handler import managementpartner_exception_handler
-
-
 def load_command_table(self, _):
-    def managementpartner_type(*args, **kwargs):
-        return CliCommandType(*args, exception_handler=managementpartner_exception_handler, **kwargs)
 
-    managementpartner_partner_sdk = managementpartner_type(
-        operations_tmpl='azext_managementpartner.managementpartner.operations.partner_operations#PartnerOperations.{}',
-        client_factory=managementpartner_partner_client_factory
-    )
+    with self.command_group('managementpartner'):
+
+        from .aaz.latest.managementpartner import Create, Delete, Update
+        self.command_table['managementpartner create'] = Create(loader=self)
+        self.command_table['managementpartner delete'] = Delete(loader=self)
+        self.command_table['managementpartner update'] = Update(loader=self)
```

Comment:
There are not required. Only customized commands required to overwrite in comands.py file.
```suggestion
    with self.command_group('managementpartner'):
```